tests/next_tests/unit_tests/embedded_tests/test_nd_array_field.py

```diff
@@ -468,10 +468,10 @@ def test_absolute_indexing_value_return():
     field = common._field(np.reshape(np.arange(100, dtype=np.int32), (10, 10)), domain=domain)
 
     named_index = ((IDim, 12), (JDim, 6))
-    value = field[named_index]
+    value = field[named_index].ndarray
```

Comment:
make sure the result is still a field
```suggestion
    assert common.is_field(field[named_index])
    value = field[named_index].ndarray
```